h/feeds/rss.py

```diff
@@ -0,0 +1,72 @@
+"""Functions for generating RSS feeds."""
+from pyramid import i18n
+from dateutil import parser
+
+from h import util
+import h.feeds.util
+
+
+_ = i18n.TranslationStringFactory(__package__)
+
+
+def _pubDate_string_from_annotation(annotation):
+    """Return a correctly-formatted pubDate string for the given annotation.
+
+    Return a pubDate string like 'Tue, 03 Jun 2003 09:39:21 GMT' from a
+    Hypothesis API 'created' datetime string like
+    '2015-03-11T10:43:54.537626+00:00'.
+
+    Suitable for use as the contents of a <pubDate> element in an <item>
+    element of an RSS feed.
+
+    """
+    return parser.parse(annotation['created']).strftime(
+        '%a, %d %b %Y %H:%M:%S %Z')
```

Comment:
This is really anal, so sorry, but you probably want `%z` here, not `%Z`. Dates in RSS should be RFC822, and `%Z` spits out stuff like "UTC" which isn't strictly RFC822-compliant.